README.md

```diff
@@ -25,37 +23,69 @@ Then start the server with gulp.
 gulp
 ```
 
+You can also use gulp with a local .env file for environmental variables.
+
+```
```

Comment:
I think lines 28 through 33 can be removed. `npm run start-local` just runs `gulp`, which line 23 already suggested

Reply:
As for line 32, I think it introduces some complexity to the instructions (check out this JSON file and then search for a "scripts" keyword in it), I don't see how this adds to the instructions though

Reply:
Fair point re: line 32, I'll delete.

For `npm run start-local` though, running gulp by itself doesn't pick up the local .env settings, which is why I added this. In my interpretation, just running `gulp` works for hot reloading on localhost, whereas `npm run start-local` does the equiv with .env overrides.

Reply:
That makes sense! Could you update line 23 to suggest `yarn start-local` then? That way we're just suggesting a single command

Reply:
Ah, thank you! Force of habit re: npm vs yarn.

Reply:
Could you move the `yarn start-local` suggestion to line 23, replacing the previous suggestion of `gulp` ? That way we're just suggesting one thing consistently